### PR TITLE
Change namespace from RubyDeps to CuttingEdge.

### PR DESCRIPTION
**Pull request by bartkamphorst**
Minor PR to change the namespace from RubyDeps to CuttingEdge. The only exception I made is in the README, where the Travis and Coverall badges depend on the github repository name, which should be changed first before changing the badges from rubydeps to cutting_edge.

### CODE REVIEW

File: bin/launch.rb [hunk 1]
```diff
@@ -40,11 +40,11 @@
 end
 
 # Need to initialize the log like this once, because otherwise it only becomes available after the Sinatra app has received a request...
-::SemanticLogger.add_appender(file_name: "#{RubyDeps.environment}.log")
+::SemanticLogger.add_appender(file_name: "#{CuttingEdge.environment}.log")
 
-RubyDeps.set(:repositories, repositories)
-RubyDeps.set(:store, store)
-RubyDeps.set(:enable_logging, true)
+CuttingEdge.set(:repositories, repositories)
```
Comment (dometto) [inline bin/launch.rb]: Perhaps this should be `CuttingEdge::App`?

File: lib/app.rb [hunk 2]
```diff
@@ -32,8 +32,8 @@ def worker_fetch(repo)
 
 
 
-class RubyDeps < Sinatra::Base
-  include RubyDepsHelpers
+class CuttingEdge < Sinatra::Base
```
Comment (dometto) [inline lib/app.rb]: Same comment as above

File: lib/workers/helpers.rb [hunk 3]
```diff
@@ -4,15 +4,15 @@
 module WorkerHelpers
```
Comment (dometto) [inline lib/workers/helpers.rb:4]: Alternatively, we could wrap this in the new namespace as well. Either works and is fine with me!

File: bin/launch.rb (hunk 1 above)
Comment (bartkamphorst) [reply bin/launch.rb]: Done.

File: lib/app.rb (hunk 2 above)
Comment (bartkamphorst) [reply lib/app.rb]: Done. The Sinatra App is now part of the module CuttingEdge, creating the namespace CuttingEdge::App.

**Review by dometto — APPROVED** [review]

**Pull request merged**